### ISSUE TITLE
Include postal code example per country/subdivision

### ISSUE THREAD

**Issue by Etroid** [commerceguys/addressing#127]
## Problem/Motivation
There are many scenarios in which the postal code validation return ambiguous error messages. It might be useful to provide some additional context through a valid example. Google provides a list of valid postal code examples under the `zipex` property. As a first step to improve postal code validation, we would need to include this property in the addressing resources.

[Example](https://i18napis.appspot.com/address/data/US)

### What's involved:
- [ ] Update `generate_address_data.php` to include `zipex` on both country and subdivision level
- [ ] Add property `postalCodeExample` to `AddressFormat` class
- [ ] ?

**Comment by jamessampford** [comment]
I think this would be a great addition as this will show an example as to what was expected by the validator

**Comment by bojanz** [comment]
I need to resolve #147 first because if we give up on subdivision-level postal code validation, then we don't need postal code examples on that level either.

**Comment by BenWalters** [comment]
@bojanz since #147 is now 'resolved' is there an appetite to implement the examples at country level?

**Comment by bojanz** [comment]
@BenWalters 
Yes, let's circle back to this. I think we need a new PR though.

**Comment by damienmckenna** [comment]
Just ran into this with the following values from El Salvador:
* 1101
* CP1101

Both failed validation, but both are "shorthand" for the address format, and in both cases it gave the vague "Postal code field is not in the right format." message.
